test/edu-benefits/pages/contactInformation.unit.spec.jsx

```diff
@@ -53,7 +53,7 @@ describe('Edu pages contactInformation', () => {
     const formDOM = findDOMNode(form);
     submitForm(form);
 
-    expect(formDOM.querySelectorAll('.usa-input-error').length).to.equal(6);
+    expect(formDOM.querySelectorAll('.usa-input-error').length).to.equal(5);
```

Comment:
I'll be honest with you; I have no idea why this worked before. The country is also a required field, but it's pre-populated with 'United States,' so that doesn't make much sense to me. There are 6 required fields, but 1 is filled in by default.